badgecheck/tasks/validation.py

```diff
@@ -211,7 +252,7 @@ def __init__(self, class_name):
                 #   'expected_class': OBClasses.VerificationObject, 'required': True},
                 {'prop_name': 'issuedOn', 'prop_type': ValueTypes.DATETIME, 'required': True},
                 {'prop_name': 'expires', 'prop_type': ValueTypes.DATETIME, 'required': False},
-                {'prop_name': 'image', 'prop_type': ValueTypes.URL, 'required': False},  # TODO: ValueTypes.DATA_URI_OR_URL
```

Comment:
There may eventually be a merge conflict here, as in my recent PR #77 I switched this TODO to a test against `ValueTypes.DATA_URI_OR_URL`.